arbor/include/arbor/cable_cell.hpp

```diff
@@ -183,6 +184,7 @@ struct cable_probe_ion_ext_concentration_cell {
 // Forward declare the implementation, for PIMPL.
 struct cable_cell_impl;
 
+
```

Comment:
The typed map definitions below can now be moved into the `.cpp` file now, I think.